### PR TITLE
Conan 0.9.4

### PR DESCRIPTION
**Pull request by lasote**
- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [X] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

Python compiled against brewed openssl is required.

### CODE REVIEW

File: Formula/conan.rb [hunk 1]
```diff
@@ -11,7 +11,12 @@ class Conan < Formula
     sha256 "aae8d317e2e9add74d893d0183c3f93539f98b81f59f67bed1d1f2a9466198bd" => :mavericks
   end
 
-  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on :python => "with-brewed-openssl"
```
Comment (MikeMcQuaid) [inline Formula/conan.rb]: Why can't it work with the system version?

**Comment by MikeMcQuaid** [comment]
> Python compiled against brewed openssl is required.

Why?

**Comment by lasote** [comment]
Because it uses an ancient OpenSSL: See the last PR #1968

File: Formula/conan.rb (hunk 1 above)
Comment (MikeMcQuaid) [inline Formula/conan.rb]: Just make this `depends_on "python"` in that case.

**Comment by MikeMcQuaid** [comment]
@lasote Why did you close and reopen this?

File: Formula/conan.rb [hunk 2]
```diff
@@ -11,7 +11,13 @@ class Conan < Formula
     sha256 "aae8d317e2e9add74d893d0183c3f93539f98b81f59f67bed1d1f2a9466198bd" => :mavericks
   end
 
-  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on :python
```
Comment (MikeMcQuaid) [inline Formula/conan.rb]: Use `depends_on "python"`, not `:python`.

**Comment by lasote** [comment]
@MikeMcQuaid I pushed empty changes and github closed automatically. My mistake.

**Comment by MikeMcQuaid** [comment]
Would like to see what @tdsmith thinks about the Python dependency but otherwise 👍

**Comment by lasote** [comment]
Any advance with this? I know you like to look carefully the formula but I'm afraid this PR overlaps with conan 0.10
Can I do something?

**Comment by MikeMcQuaid** [comment]
@lasote I'd like to give @tdsmith more than a day to check in, thanks.

**Comment by tdsmith** [comment]
Please install the set of packages that the `requests[security]` option installs, including `cryptography` and `pyOpenSSL`; `cryptography` will link against Homebrew's openssl and allow requests to speak modern TLS even with the system Python.

**Comment by lasote** [comment]
Sorry, but I don't understand, pyOpenSSL and Cryptography packages are already installed, or not?

```
+  resource "cryptography" do
 +    url "https://pypi.python.org/packages/a9/5b/a383b3a778609fe8177bd51307b5ebeee369b353550675353f46cb99c6f0/cryptography-1.4.tar.gz"
 +    sha256 "bb149540ed90c4b2171bf694fe6991d6331bc149ae623c8ff419324f4222d128"
 +  end
```

```
+  resource "pyOpenSSL" do
 +    url "https://pypi.python.org/packages/77/f2/bccec75ca4280a9fa762a90a1b8b152a22eac5d9c726d7da1fcbfe0a20e6/pyOpenSSL-16.0.0.tar.gz"
 +    sha256 "363d10ee43d062285facf4e465f4f5163f9f702f9134f0a5896f134cbb92d17d"
 +  end
```

**Comment by tdsmith** [comment]
Can you show me how to provoke a failure using system Python?

**Comment by lasote** [comment]
Yes, this command downloading openssl tgz should fail:

> conan install OpenSSL/1.0.2h@lasote/stable --build OpenSSL

**Comment by tdsmith** [comment]
Oh, sorry, you also need to `touch libexec/"vendor/lib/python2.7/site-packages/ndg/__init__.py"` because everything is bad.

**Comment by lasote** [comment]
Updated!

**Comment by dunn** [comment]
Test is failing: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/3662/version=yosemite/testReport/junit/brew-test-bot/yosemite/test_conan/

**Comment by lasote** [comment]
@dunn thanks for notice. I've added the setuptools. Hope now works.

**Comment by lasote** [comment]
do you need any other change?

**Comment by dunn** [comment]
I'll let @tdsmith give the final sign-off.

**Comment by lasote** [comment]
@tdsmith sorry for the ping but I would need to resolve this asap... :(

**Comment by tdsmith** [comment]
Sorry for the delay!

**Pull request closed without merging**